odhlasit.php

```diff
@@ -0,0 +1,6 @@
+<?php
+session_start();
+if (isset($_GET['odhlasit']) && $_GET['odhlasit'] == '1') {
```

Comment:
a musi tady byt vubec ta podminka? nemuze to proste jen udelat session_destroy a redirect?